angular.json

```diff
@@ -71,8 +72,12 @@
               "outputHashing": "all"
             },
             "development": {
-              "buildOptimizer": false,
-              "optimization": false,
+              "verbose": true,
+              "aot": true,
+              "statsJson": true,
+              "showCircularDependencies": true,
+              "buildOptimizer": true,
+              "optimization": true,
```

Comment:
in development you might still want these as `false` for easier debugging

Reply:
valid, but i dont like how dev and prod builds will differ (with these being enabled by default in prod) so i left them in. i can remove if you dont like that

Reply:
Just seems weird to minify / tree shake / optimize in development. Maybe add a `"build:prod": ng build --prod` script to the package.json, that way it's clear how to run and check for a prod build. Feel free to resolve if you disagree